app/views/admin/organizations/_form.html.slim

```diff
@@ -0,0 +1,66 @@
+= simple_form_for(@coop,
+  url: admin_organization_path,
+  html:{ \
+    class: 'form-horizontal organization-record',
+    data: {'id': @coop.id, 'method': 'PUT'},
+  },
+  defaults: { \
+    input_html: { class: 'form-element col-lg-6' },
+  },
+  method: :put) do |f|
+
+  div.form-group.row
+    label.col-lg-3 Name
+    span.col-lg-6.view-element
+      = @coop.name
+    = f.input_field :name
+
+  div.form-group.row
+    label.col-lg-3 Address
+    span.col-lg-6.view-element
+      = @coop.street_address
+    = f.input_field :street_address, as: :text
+
+  div.form-group.row
+    label.col-lg-3 City
+    span.col-lg-6.view-element
+      = @coop.city
+    = f.input_field :city
+
+  div.form-group.row
+    label.col-lg-3 State
+    span.col-lg-6.view-element
+      = @coop.state
+    = f.input_field :state
+
+  div.form-group.row
+    label.col-lg-3 Country
+    - unless @coop.new_record?
+      span.col-lg-6.view-element
+        = @coop.country.name
+    = select(:coop, :country_id) do
+      - @countries.each do |country|
+        = content_tag(:option, country.name, value: country.id)
+
+  div.form-group.row
+    label.col-lg-3 Website
+    span.col-lg-6.view-element
+      - unless @coop.website.nil?
+        = link_to(@coop.website, @coop.website)
+    = f.input_field :website
+
+  div.form-group.row
```

Comment:
This isn't fully set up yet until we have people in database. And adding members not set up yet.